buildSrc/src/main/groovy/io/deephaven/tools/docker/DeephavenInDockerExtension.groovy

```diff
@@ -105,13 +114,49 @@ public abstract class DeephavenInDockerExtension {
             }
         }
 
+        logTask = project.tasks.register('writeServerLogs', DockerLogsContainer) { task ->
+            task.targetContainerId containerName.get()
+
+            task.onlyIf = Specs.union(
```

Comment:
I started there, but we would need to unconditionally write that as an output, and since there are timestamps in outputs, that gets gross, they are never up to date.

Or perhaps worse, _not_ declare it as an output, but I think gradle gets grumpy if it notices you do this.

All of these server logs are drowned in the noise of a normal nightly run, we'd barely even notice them, they are quite small.